CHANGELOG.md

```diff
@@ -26,6 +26,13 @@ Other major additions
 Other minor additions
 ---------------------
 
+* Added new proofs to `Induction.WellFounded`:
+  ```agda
+  someWfRecInvar : ∀ x → (q q' : Acc _<_ x) → Some.wfRec P f x q ≡ Some.wfRec P f x q'
+  wfRecBuilder-wfRec : ∀ {x y} y<x → All.wfRecBuilder P f x y y<x ≡ All.wfRec P f y
+  wfRecFixpoint : ∀ {x} → All.wfRec P f x ≡ f x λ y _ → All.wfRec P f y
+  ```
```

Comment:
We usually drop the implicit + not super interesting arguments in the CHANGELOG.
I would expect something like this:

```agda
  someWfRecInvar     : Some.wfRec P f x q ≡ Some.wfRec P f x q'
  wfRecBuilder-wfRec : All.wfRecBuilder P f x y y<x ≡ All.wfRec P f y
  wfRecFixpoint      : All.wfRec P f x ≡ f x λ y _ → All.wfRec P f y
```